cli/src/installers/envVars.ts

```diff
@@ -9,6 +9,7 @@ export const envVariablesInstaller: Installer = async ({
 }) => {
   const usingAuth = packages?.nextAuth.inUse;
   const usingPrisma = packages?.prisma.inUse;
+  const usingTrpc = packages?.trpc.inUse;
```

Comment:
this is not needed. we have a default schema in template/base which we should use instead.

Reply:
Ok, I'll do that instead.